project/src/main/java/com/google/sps/data/ServletLibrary.java

```diff
@@ -0,0 +1,165 @@
+package com.google.sps.data;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.vision.v1.*;
+
+import java.util.*;
+ 
+import java.io.IOException;
+
+public class ServletLibrary{
```

Comment:
Just as a note for merging, could you change the name of this file as I have one also called this, but the purpose is entirely different(for me its datastore stuff). You could call it ProductSearchLibrary or something. I will probably be changing the name of mine to be something that better explains what the purpose of the file is.